src/Umbraco.Core/Persistence/Repositories/Implement/ContentTypeRepositoryBase.cs

```diff
@@ -527,20 +505,18 @@ protected void PersistUpdatedBaseContentType(IContentTypeComposition entity)
         /// <summary>
         /// Ensures that no property types are flagged for a variance that is not supported by the content type itself
         /// </summary>
-        /// <param name="entity"></param>
+        /// <param name="entity">The entity for which the property types will be validated</param>
         private void ValidateVariations(IContentTypeComposition entity)
         {
-            //if the entity does not vary at all, then the property cannot have a variance value greater than it
-            if (entity.Variations == ContentVariation.Nothing)
+            foreach (var prop in entity.PropertyTypes)
             {
-                foreach (var prop in entity.PropertyTypes)
-                {
-                    if (prop.IsPropertyDirty(nameof(prop.Variations)) && prop.Variations > entity.Variations)
-                        throw new InvalidOperationException($"The property {prop.Alias} cannot have variations of {prop.Variations} with the content type variations of {entity.Variations}");
-                }
```

Comment:
This validate seemed incomplete due to the `ContentVariation.Nothing` and the `prop.IsPropertyDirty()` check, it did not match with the description of the method. For example, it would allow a property type to have ContentVariation.Culture when the content type was set to ContentVariation.Nothing, as long as the property type was not dirty. This does not seem to make much sense, it should not be allowed at all regardless of the dirty state.

It also allowed various other invalid combinations, like content type of Culture and property type of Segment.

Reply:
Yep, see my last comment, maybe this is actually unecessary anyways if we are auto adjusting?